.changeset/beige-cups-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+This PR introduces upates to the category names in the Flight library
```

Comment:
We don't use language like "This PR introduces..." in the changesets. What about something more descriptive, like: 

> Reorganized the icons within the "Core" icon set into 16 more descriptive categories.

Note our Changelog content best practices: https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#content-formatting